packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -74,24 +72,75 @@ function clientExtensionsModelDefinition(this: PrismaClientClass) {
 function clientExtensionsQueryDefinition(this: PrismaClientClass) {
   const modelNames = Object.keys(this.dmmf.getModelMap())
 
-  const params = `{${modelNames.reduce((acc, modelName) => {
-    const actions = getModelActions(this.dmmf, modelName)
+  const prismaNamespaceTypes = `type QueryExtensionArgs<ExtArgs extends runtime.Types.Extensions.Args = { result: {}, model: {}, query: {}, client: {} }> = {${modelNames.reduce(
```

Comment:
A special object type that exposes input and output types for a give model + action. Useful for typing `query`.